src/recipes/ios-promo/index.js

```diff
@@ -108,13 +108,6 @@ let waitedEnough = function (restDays, last, now) {
 };
 
 
```

Comment:
I would probably leave it in.  It's dead code though.  I assume we are 10% everywhere?  That's our solution?